api/features/views.py

```diff
@@ -636,6 +637,31 @@ def all(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @swagger_auto_schema(request_body=IdentitySourceIdentityRequestSerializer())
+    @action(methods=["POST"], detail=False, url_path="clone-from-given-identity")
+    def clone_from_given_identity(self, request, *args, **kwargs) -> Response:
```

Comment:
nit: missing typing in the args

Reply:
There's a ton of code that sets `request: Request` and our orders from @matthewelwell is to always set the typing even in places that seem silly, like typing in test fixtures.